src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+import javax.lang.model.type.NullType;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Manager extends AbstractEmployee {
+
+    protected List<Employee> employees;
+
+
+
+    public Manager(String name, double salary) {
+        super(name, salary);
+        this.employees=new ArrayList<Employee>();
+    }
+
+    public Employee getEmployeeByName(String name){
+        Employee none =new Employee("None",0);
+        for(Employee x: employees){
+            if(x.getName()==name) return x;
+        }
+        return none;
```

Comment:
Ta metodka powinna zwracać null'a jeśli nie znajdzie `Employee`.
Jest jeszcze parę innych sposobów jeśli chce się zwracania null'a uniknąć, ale w taki sposób raczej tego nie robimy :P

Reply:
Chciałem, żeby zwracała NULL'a ale nie wiem w jaki sposób to zapisać
Używając "return NULL" dostawałem błąd...

Reply:
`return null;` z małych literek w javie 😄